plugin/trino-ignite/src/test/resources/ignite.sh

```diff
@@ -0,0 +1,230 @@
+#!/usr/bin/env bash
```

Comment:
Do you need to pass the entire file here? What have you modified?

Reply:
Mainly modified is related the Ignite server and jvm config:
1. I have changed the Ignite ``DIGNITE_SQL_MERGE_TABLE_MAX_SIZE`` for the Ignite to allow the server return larger size of data.
2. Modified the script to allow the version 2.8 of Ignite can run on JDK17(--add-opens some module)
So custom the initial script here, any suggestions for better making it for your side?

Reply:
Ad. 1. Is changing `DIGNITE_SQL_MERGE_TABLE_MAX_SIZE ` required? Maybe you can pass the value of that env variable externally?

Changing the entire file is problematic, as it will cause issues whenever one would like to update the version of Ignite. Can we somehow avoid overriding Ignite product code? Can we fix this in Ignite? Maybe never version of Ignite supports JDK17?

Reply:
Yes, for the test like ``testDynamicFilteringWithAggregationGroupingColumn`` would raise ``Fetched result set was too large`` exception from IgniteServer if not adjust the param. Will make this by set env, thanks for the suggestion.